osu.Game/Overlays/Settings/Sections/Input/KeyBindingsSubsection.cs

```diff
@@ -67,7 +67,7 @@ private void load()
         {
             Text = InputSettingsStrings.ResetSectionButton;
             RelativeSizeAxes = Axes.X;
-            Width = 0.5f;
+            Width = Content.Width;
```

Comment:
this line can actually be removed entirely (by default drawables have a width of 1, which combined with relative-sized X axes will yield the same result).

transferring `Width` from arbitrary drawables it not generally valid. whether it is depends on relative sizing.

Reply:
I intentionally made is less wide as a design choice. May be okay with bumping it up slightly, but generally when it comes to buttons (where language is intended to be concise), I'd say localisations should be reworded to fit the UI, not vice-versa.

What's the minimum width required for the localisation to fit, and before answering that, can it be reworded to be more concise?

Reply:
The widest is Spanish, which is almost full width. I don't speak Spanish but I'll make an effort to find someone who can.

Reply:
Localisers need to be aware of UI limitations when localising. A note can be made on crowdin to warn from using excessive string lengths.

Reply:
I believe crowdin has a character counter in its string editor, and it does light up if the translation is significantly longer than the original. It does require translators to be aware of what that means, though.

Reply:
So what are you suggesting? Making the width .75f ish, and then limiting the translatio to fit?

Reply:
> I believe crowdin has a character counter in its string editor, and it does light up if the translation is significantly longer than the original. It does require translators to be aware of what that means, though.

Interestingly though, it lights up when the localised text characters count has double the count of the original text characters, so it might be unhelpful in some cases. Not sure if it's configurable though.

Reply:
`0.75f` seems like a better value, yep (in fact, `0.8f` is also fine). I'm going to change to that and we can deal with the localisation elsewhere if it's still not fitting.